jangjuseong/db/migrations/20230120021617_users.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE users (
+id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+name VARCHAR(50) NOT NULL,
+email VARCHAR(200) NOT NULL,
+password VARCHAR(200) NOT NULL,
+profile_image VARCHAR(1000) NULL,
+created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+CONSTRAINT users_email_ukey UNIQUE(email)
+);
```

Comment:
이 부분 indentation (들여쓰기) 적용해서 가독성을 높여주면 좋을 것 같습니다!

추가로, 나중에 비밀번호를 암호화하여 저장을 할 텐데, 암호화 된 비밀번호는 varchar() 보다는 binary() 값으로도 저장할 수도 있습니다.
mysql 공식문서에서 더 많은 타입을 찾아보고 적용해보면 좋을 것 같습니다!!


```suggestion
CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    name VARCHAR(50) NOT NULL,
    email VARCHAR(200) NOT NULL,
    password VARCHAR(200) NOT NULL,
    profile_image VARCHAR(1000) NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
    CONSTRAINT users_email_ukey UNIQUE(email)
);
```